packages/firebase/firestore/memory/index.ts

```diff
@@ -21,4 +21,4 @@
  */
 
 import '@firebase/firestore/memory';
-import '@firebase/firestore/bundle';
+import '@firebase/firestore/memory-bundle';
```

Comment:
Can this be `@firebase/firestore/memory/bundle`?

Reply:
It cannot be done gracefully. 

The reason is relative paths in `packages/firestore/memory-bundle/package.json` are interpreted in different context (in rollup or in Typescript import from packages/firebase/firestore/...) and therefore some of them need to be prefixed `../..` and some of them just `../`, which is confusing.